feedback/static/html/studio_view.html

```diff
@@ -15,75 +15,84 @@
 
     <li class="field comp-setting-entry is-set">
       <div class="wrapper-comp-setting">
-        <label class="label setting-label" for="freeform">Freeform prompt</label>
-        <input class="input setting-input" name="freeform" id="freeform" value="{freeform}" type="text" />
+        <label class="label setting-label">Freeform prompt
+          <input class="input setting-input" name="freeform" id="freeform" value="{freeform}" type="text" />
```

Comment:
Nit: The thing that's nice about doing the nesting is that you then do not need the IDs (id="freeform"), assuming they aren't being used elsewhere. And in general, we avoid IDs.